examples/fourier_transform_playground.py

```diff
@@ -140,12 +139,15 @@ def moving_wave(
         thread.send((wave_id, frequency, angle, phase_shift, speed))
 
 
-wdg = moving_wave()
-
 # add the widget to the window and run it once
-viewer.window.add_dock_widget(wdg, area='bottom')
-wdg()
-
-napari.run()
+viewer.window.add_dock_widget(moving_wave, area='bottom')
+moving_wave()
+
+if __name__ == '__main__':
+    napari.run()
+else:
+    # for the gallery, sleep to make sure that the images are captured
+    from time import time
```

Comment:
```suggestion
    import time
```
You had it correct before.
time.time is a different method:
https://docs.python.org/3/library/time.html#time.time

Or do `from time import sleep` and use `sleep(5)`

Reply:
🤦 

Thanks @psobolewskiPhD!!!

Reply:
Oooh shit I misunderstood why it failed before:

```python
  ________________________________________________________________________________
  Exceptions caught in Qt event loop:
  ________________________________________________________________________________
  Traceback (most recent call last):
    File "/home/runner/work/napari/napari/examples/fourier_transform_playground.py", line 70, in combine_and_set_data
      t = time()
  TypeError: 'module' object is not callable
```

Looks like there is a name scoping issue when I import time later! So from time import sleep it is!